linode/manager
# M3-6053: Standardize usage of dismissible banners and notices

## Description 📝
- Remove unused/old banners
- Refactor Notice component to accept a dismissible button prop
- Refactor DismissibleBanner component to use the Notice component
    - This allows us to use the same NoticeProps to customize the banner instead of redefining the same props
    - Also standardizes how we use dismissible banners/notices
- Refactor ProductInformationBanner component to leverage the above
- Update Banner and Notice stories in Storybook

## How to test 🧪
Check the banners/notices changed in this PR

run `yarn storybook` and check the updated stories:
- http://localhost:6006/?path=/docs/components-notifications-dismissible-banners--example-beta-banner
- http://localhost:6006/?path=/docs/components-notifications-notices--success
